ci/workflows/flink-bench.yml

```diff
@@ -0,0 +1,12 @@
+steps:
+  - label: "flink nexmarkbenchmark"
+    command: "ci/scripts/flink-bench.sh"
+    plugins:
+      - ./ci/plugins/benchmark
+      - seek-oss/aws-sm#v2.3.1:
+          env:
+            GITHUB_TOKEN: github-token
+            BENCH_TOKEN: bench-token
+    agents:
+      queue: "flink-bench"
```

Comment:
"flink-bench" should not be used here. The queue here represents the buildkite stack. We currently only have two buildkite stacks, one for benchmark workflow and one for other workflows.

You can also use "benchmark" queue to run flink bench.